docusaurus/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Dependencies
```

Comment:
Commit message: Wasn't "docusaurus" instead of "my-website" used? Because I do not see a "my-website" directory being created.